adapters/httpapi_compact.c

```diff
@@ -731,10 +735,11 @@ static HTTPAPI_RESULT OpenXIOConnection(HTTP_HANDLE_DATA* http_instance)
             }
             else
             {
+				int countRetry;
```

Comment:
Discussed this in person: if really desired this should be gated. Otherwise it simply creates noise in reviews :-).